src/utils/AnalyzeFLASHIda.cpp

```diff
@@ -0,0 +1,454 @@
+// For a full list of authors, refer to the file AUTHORS.
+// --------------------------------------------------------------------------
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL ANY OF THE AUTHORS OR THE CONTRIBUTING
+// INSTITUTIONS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+// OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+// WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+// ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// --------------------------------------------------------------------------
+// $Maintainer: Kyowon Jeong $
+// $Authors: Kyowon Jeong $
+// --------------------------------------------------------------------------
+
+#include <OpenMS/APPLICATIONS/TOPPBase.h>
+#include <OpenMS/FORMAT/MzMLFile.h>
+#include <OpenMS/KERNEL/MSExperiment.h>
+#include <OpenMS/SYSTEM/File.h>
+#include <OpenMS/FORMAT/FileHandler.h>
+
+#include <QFile>
+#include <iomanip>
+
+#include <OpenMS/METADATA/SpectrumLookup.h>
+using namespace OpenMS;
+using namespace std;
+
+//-------------------------------------------------------------
+// Doxygen docu
+//-------------------------------------------------------------
+
+/**
+    @page
+*/
+
+/// @cond
+
+class TOPPAnalyzeFLASHIda :
+    public TOPPBase
+{
+public:
+  TOPPAnalyzeFLASHIda() :
+      TOPPBase("AnalyzeFLASHIda", ".", false)
+  {
+  }
+
+protected:
+  void registerOptionsAndFlags_() override
+  {
+    //registerOutputFileList_("out_spec", "<file for MS1, file for MS2, ...>", {""},
+    //                            "output files (tsv) - spectrum level deconvoluted masses per ms level", false);
+    //
+
+    registerInputFileList_("in", "<files >", {}, "Input files");
+    registerOutputFile_("out", "<file>", "", "");
+    //setValidFormats_("i1n", ListUtils::create<String>("tsv"));
+    //setValidFormats_("i1n", ListUtils::create<String>("tsv"));
+  }
+
+  struct TopPicItem
+  {
+  public:
+    TopPicItem() = default;
+
+    TopPicItem(String in)
+    {
+      str_ = in;
+      vector<String> results;
+      stringstream tmp_stream(in);
+      String str;
+
+      while (getline(tmp_stream, str, ','))
+      {
+        results.push_back(str);
+      }
+      //cout<<str_<<endl;
+      protein_acc_ = results[0];
+      proteform_id_ = stoi(results[1]);
+      rt_ = stod(results[2]);
+      precursor_mass_ = stod(results[4]);
+      precursor_mz_ = stod(results[5]);
+      precursor_intensity = stod(results[6]);
+        mass_intensity_ = stod(results[7]);
```

Comment:
additional whitespace

Reply:
Timo - I will remove these files - please ignore those in src/utils